recipes/r-robma/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set version = '1.2.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-robma
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RoBMA_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RoBMA/RoBMA_{{ version }}.tar.gz
+  sha256: 0cb6e8eec07a81b42ce00315a94d03a6617828850c0a446b512f8576d26cd331
+
+build:
+  skip: True  # [not linux]
```

Comment:
Please add a short comment above the skip on why osx/win are being skipped.

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/15555/commits/b32795f04a20b0e2a48cc00e3db1933d9c5703b1 , thank you!